docs/getting-started/publish-infrastructure.md

```diff
@@ -0,0 +1,506 @@
+    names:
+      kind: MySQLInstance
+      listKind: MySQLInstanceList
+      plural: mysqlinstances
+      singular: mysqlinstance
+    validation:
+      openAPIV3Schema:
+        type: object
+        properties:
+          spec:
+            type: object
+            properties:
+              parameters:
+                type: object
+                properties:
+                  storageGB:
+                    type: integer
+                required:
+                  - storageGB
+            required:
+              - parameters
+```
+
+You are now able to create instances of kind `MySQLInstance` at the cluster
+scope.
+
+## Create InfrastructurePublication
+
+The `InfrastructureDefinition` will make it possible to create `MySQLInstance`
+objects in our Kubernetes cluster at the cluster scope, but we want to make them
+available at the namespace scope as well. This is done by defining an
+`InfrastructurePublication` that references the new `MySQLInstance` type.
+
+```yaml
+apiVersion: apiextensions.crossplane.io/v1alpha1
+kind: InfrastructurePublication
+metadata:
+  name: mysqlinstances.database.example.org
+spec:
+  infrastructureDefinitionRef:
+    name: mysqlinstances.database.example.org
+```
+
+This will create a new CRD named `MySQLInstanceRequirement`, which is the
+namespace-scoped variant of the `MySQLInstance` CRD. You are now able to create
+instances of kind `MySQLInstanceRequirement` at the namespace scope.
+
+## Create Compositions
+
+Now it is time to define the resources that represent the primitive
+infrastructure units that actually get provisioned. For each provider we will
+define a `Composition` that satisfies the requirements of the `MySQLInstance`
+`InfrastructureDefinition`. In this case, each will result in the provisioning
+of a public MySQL instance on the provider.
+
+
+<ul class="nav nav-tabs">
+<li class="active"><a href="#aws-tab-1" data-toggle="tab">AWS</a></li>
+<li><a href="#gcp-tab-1" data-toggle="tab">GCP</a></li>
+<li><a href="#azure-tab-1" data-toggle="tab">Azure</a></li>
+<li><a href="#alibaba-tab-1" data-toggle="tab">Alibaba</a></li>
+</ul>
+<br>
+<div class="tab-content">
+<div class="tab-pane fade in active" id="aws-tab-1" markdown="1">
+```yaml
+apiVersion: apiextensions.crossplane.io/v1alpha1
+kind: Composition
+metadata:
+  name: mysqlinstances.aws.database.example.org
+  labels:
+    provider: aws
+spec:
+  writeConnectionSecretsToNamespace: crossplane-system
+  reclaimPolicy: Delete
+  from:
+    apiVersion: database.example.org/v1alpha1
+    kind: MySQLInstance
+  to:
+    - base:
+        apiVersion: database.aws.crossplane.io/v1beta1
+        kind: RDSInstance
+        spec:
+          forProvider:
+            dbInstanceClass: db.t2.small
+            masterUsername: masteruser
+            allocatedStorage: 20
+            engine: mysql
+            skipFinalSnapshotBeforeDeletion: true
+            publiclyAccessible: true
+          writeConnectionSecretsToRef:
```

Comment:
```suggestion
          writeConnectionSecretToRef:
```

haven't tested AWS yet, but docs say so this:
https://doc.crds.dev/github.com/crossplane/provider-aws/config/crd/database.aws.crossplane.io_rdsinstances.yaml#spec-writeConnectionSecretToRef